apps/nextjs-website/src/components/templates/GitBookTemplate/GitBookTemplate.tsx

```diff
@@ -34,7 +35,9 @@ const GitBookTemplate = ({
   breadcrumbs,
   menuDistanceFromTop,
   contentMarginTop = 75,
+  hasHeader = true,
 }: GitBookTemplateProps) => {
+  const headerPadding = hasHeader ? '60px' : '-80px';
```

Comment:
```suggestion
  const paddingTop = hasHeader ? '60px' : '-80px';
```
or `headerPaddingTop` or an object containing paddingTop and paddingX